### PR TITLE
Added .NET Core to the project list

### PR DESCRIPTION
**Pull request by shiftkey**
They're using the tag, so why not add it here? :trollface:

@davkean @terrajobst let me know if you have a better suggestion for the description field.

### CODE REVIEW

**Comment by shiftkey** [comment]
There's an `accepting-prs` label which [appears to represent the same thing](https://github.com/dotnet/corefx/labels/accepting%20prs) - should this be replaced?

**Comment by terrajobst** [comment]
Thanks! I meant to create the PR myself but lost track of it.

`Accepting PRs` is marking [API proposals](https://github.com/dotnet/corefx/wiki/API-Review-Process) that passed our review. Could be considered the same, though. [I've started a thread](https://github.com/dotnet/corefx/issues/455).

File: _data/projects.yml [hunk 1]
```diff
@@ -724,3 +724,11 @@
   upforgrabs:
     name: help out!
     link: https://github.com/kellabyte/Haywire/labels/help%20out!
+    
+- name: .NET Core
+  desc: The class libraries for the next version of the .NET Framework
```
Comment (terrajobst) [inline _data/projects.yml]: That's a bit misleading as .NET Core is a self-contained stack. Some of the API will be ported to the .NET Framework, some will not. What about: `Contains the foundational libraries that make up the .NET Core development stack`

Comment (shiftkey) [inline _data/projects.yml]: :ok_hand: I wrote it in haste, not attached to it at all

Comment (terrajobst) [inline _data/projects.yml]: Sweet! Thanks a lot for doing this!

**Pull request merged**